Automattic/jetpack
# admin-ui: Register Akismet menu for Jetpack-the-plugin too

## Proposed changes:

In #32911, we changed Jetpack to try to start using Admin_Menu to register a menu for Akismet instead of doing it itself, to avoid a bug where we'd get duplicate menu entries if a standalone plugin initialized Admin_Menu before Jetpack got loaded.

Unfortunately it wasn't noticed that Admin_Menu specifically avoids registering a menu for Akismet if Jetpack is present, so now Akismet isn't being registered at all.

Fix that by removing that check, allowing Admin_Menu to register Akismet for Jetpack too.

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack/issues/33550#issuecomment-1757583709
Fixes #33550

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Install and connect Jetpack. Purchase a plan that bundles Akismet. Ensure the "Akismet Anti-Spam" menu is present, and that you can successfully visit `/wp-admin/admin.php?page=akismet-key-config`.
* Install and activate Jetpack Backup. Ensure that only one Anti-Spam menu is present.
